contracts/implementations/nodes/AftermarketDevice.sol

```diff
@@ -246,6 +246,86 @@ contract AftermarketDevice is
         emit AftermarketDeviceClaimed(aftermarketDeviceNode, owner);
     }
 
+    /// @notice Pairs an aftermarket device with a vehicle through a metatransaction.
+    /// Both vehicle and AD owners can pair.
+    /// The vehicle/AD owner signs a typed structured (EIP-712) message in advance and submits to be verified
+    /// @dev Caller must have the admin role
+    /// @param aftermarketDeviceNode Aftermarket device node id
+    /// @param vehicleNode Vehicle node id
+    /// @param vehicleOwnerSig Vehicle owner signature hash
+    /// @param aftermarketDeviceSig Aftermarket Device's signature hash
```

Comment:
Order switched? I do note that in the claiming function it's ordered `aftermarketDeviceOwnerSig, aftermarketDeviceSig`. This is a different scenario, obviously.

Reply:
I know, I just think it makes more sense switching it bc we have AD node first, since we are pairing an AD. I also wanted to switch in the claiming function too, but I was afraid to ruin the backend integration and set fire to everything.